package.json

```diff
@@ -30,6 +30,7 @@
     "react-bootstrap-table-next": "^3.3.5",
     "react-dom": "^16.12.0",
     "react-simple-maps": "^1.0.0-beta.0",
+    "react-switch": "^5.0.1",
```

Comment:
Why introduce this new dependency? Is it just a toggle? Why not use the react-bootstrap one we already have: https://react-bootstrap.github.io/components/forms/#forms-custom-switch